### ISSUE TITLE
Suppress message: "Download the React DevTools for ..."

### ISSUE THREAD

**Issue by RRMoelker** [facebook/react#12041]
**feature/bug**

**What is the current behavior?**
React library logs either "*Download the React DevTools for a better development experience: https://fb.me/react-devtools*" or logs "*Warning: The installed version of React DevTools is too old and will not work with the current version of React. Please update React DevTools. https://fb.me/react-devtools*" if previous "fix" is employed.

**reproduce**
Run React site without production flag and without React DevTools extension installed.

**What is the expected behavior?**
No log output.

**Which versions of React**

React: 16.2.0

**Tried**
According to comment on #3877 the following should work in React >= 16.1.0.
```
      new webpack.DefinePlugin({
        '__REACT_DEVTOOLS_GLOBAL_HOOK__': '({ isDisabled: true })'
      }),
```
However this changes the log to:
"Warning: The installed version of React DevTools is too old and will not work with the current version of React. Please update React DevTools. https://fb.me/react-devtools"

Other fixes such as those from StackOverflow yield the same results: https://stackoverflow.com/a/42196820.

Also the plugin itself doesn't work for me because of https://github.com/facebook/react-devtools/issues/897. So installing it isn't an option, for me at least.

**TLDR**

I'm loving React. But logging the install note is a tad annoying. Previous fixes don't seem to work. So please provide an option to disable the log.

**Comment by gaearon** [comment]
>Also the plugin itself doesn't work for me because of facebook/react-devtools#897. So installing it isn't an option, for me at least.

You can star [this issue](https://bugs.chromium.org/p/chromium/issues/detail?id=778443&can=2&start=0&num=100&q=&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified&groupby=&sort=) and ask the Chrome team to fix it. There's nothing we can do unfortunately, it's caused by a Chrome regression.

>However this changes the log to:
>"Warning: The installed version of React DevTools is too old and will not work with the current version of React. Please update React DevTools. https://fb.me/react-devtools"

You can see from the source code that [the condition should exit before that log](https://github.com/facebook/react/blob/bd6b533c29cc617b4ddfc0a8b623e782c468cd52/packages/react-reconciler/src/ReactFiberDevToolsHook.js#L39-L45) if `isDisabled` is `true`.


I just created a project with Create React App, ejected it, and then added

```js
      new webpack.DefinePlugin({
        '__REACT_DEVTOOLS_GLOBAL_HOOK__': '({ isDisabled: true })'
      }),
```

to the configuration. It removed the log for me.

So we'll need a reproducing project if you say it doesn't work.

**Comment by RRMoelker** [comment]
Wow, quick response. I'll try to get a reproducible project up tomorrow.

**Comment by gaearon** [comment]
Thanks!

**Comment by gaearon** [comment]
Ah okay. For the reference, we always recommend matching versions of all React packages.

**Comment by ty-everett** [comment]
Any way to get rid of this message in RollupJS?
